d1/main/gauges.c

```diff
@@ -2376,7 +2378,7 @@ void hud_show_kill_list()
 		if (Players[player_num].connected != CONNECT_PLAYING) {
```

Comment:
Is this going to be a problem if `Show_kill_list == 3` and the first or second player happens to be disconnected?

Reply:
Good catch, I'll add a check for that.